qemu/tests/cfg/hotplug_virtio_mem.cfg

```diff
@@ -0,0 +1,36 @@
+- hotplug_virtio_mem:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = hotplug_virtio_mem
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    smp = 8
+    vcpu_maxcpus = ${smp}
```

Comment:
I think we can remove these two lines,  avocado-vt  will get a value automatically based on the host . In my opinion,  we don't have to keep the same value with the polarion case here.

Reply:
For the arm platform, since we do not support vcpu hot plugging, we need to keep vcpu_maxcpus = ${smp}. If other platforms won't need it, I think it can be written as:
```
aarch64:
    vcpu_maxcpus = ${smp}
```

Reply:
@zhenyzha @menli820 Included the latest proposal, but should I have included `smp = 8` under `aarch64:` too?

Reply:
> vcpu_maxcpus = ${smp}

yes , I think so, or it will pop up

> @zhenyzha @menli820 Included the latest proposal, but should I have included `smp = 8` under `aarch64:` too?

yes, I think so , or it will pop up error.